client/src/common_text/common_questions.yaml

```diff
@@ -0,0 +1,21 @@
+common_faq_q:
```

Comment:
So there's another file named the exact same. I think file should be renamed to something like ` common_faq_questions`. Also I think this would fit better along with faq in directory structure